src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- <link rel="stylesheet" href="./styles/correct.css"> -->
+    <link rel="stylesheet" href="https://unicons.iconscout.com/release/v3.0.1/css/line.css">
```

Comment:
Kindly remove the unwanted line of code to enhance the readability of your code

Reply:
![image](https://user-images.githubusercontent.com/90483434/180234075-61bd6462-0ef2-4256-8237-3731b68d3bcf.png)